### PR TITLE
Fix contrib note (based from #4241)

### PR DESCRIPTION
**Pull request by chemelnucfin**
I addressed the comments in #4241, so if I may pull the trigger on that, someone can look at the diff here.

### CODE REVIEW

**Comment by mfschwartz** [comment]
@chemelnucfin - your updates LGTM, though it looks like @lukesneeringer is the requested reviewer.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -62,8 +62,26 @@ but you can also use a ``virtualenv`` of your own creation.
 
 .. _repo: https://github.com/GoogleCloudPlatform/google-cloud-python
 
-Using a custom ``virtualenv``
-=============================
+Use ``nox``, not ``virtualenv``
```
Comment (jonparrott) [inline CONTRIBUTING.rst]: Can someone explain why we need this section at all? There's instructions below on how to use Nox.

Comment (mfschwartz) [reply CONTRIBUTING.rst]: +1
Using virtualenv directly creates problems, so I think we should just say to use nox, and drop this section.

**Comment by chemelnucfin** [comment]
@jonparrott Removed virtualenv section

**Pull request merged**